digits/templates/job.html

```diff
@@ -235,6 +235,53 @@ <h4 class="panel-title">
         </div>
         <div class="gpu-utilization-info" style="display:none;"></div>
 
+
+        <!-- Related jobs -->
+        {% if related_jobs is not none and related_jobs|length > 0 %}
+        <div class="panel panel-default">
+            <div class="panel-heading">
+                <h4 class="panel-title">
+                    Related jobs
+                </h4>
+            </div>
+            <div class="panel-collapse">
+                <div class="panel-body">
+                    <div class="panel-group">
+                        <!-- Related model jobs -->
+                        {% for r_job in related_jobs %}
+                        {% if prev_job_model != r_job.job_type() %}
+                            <h4 class="panel-title" style="margin:10px">{{r_job.job_type()}}</h4>
+                            {% set prev_job_model = r_job.job_type() %}
+                        {% endif %}
+                        <div id="{{r_job.id()}}-status" class="panel panel-{{ r_job.status.css }}">
+                            <div class="panel-heading">
+                                <h4 class="panel-title">
+                                    <a target="_blank" href="/jobs/{{ r_job.id() }}">
```

Comment:
Something like this could have worked too:
```
<a href="{{ url_for('digits.views.show_job', job_id=r_job.id()) }}">
```
I am not sure which is better... I tend to use this form. I feel it's more resilient to future changes.